app/liquid_tags/jsitor_tag.rb

```diff
@@ -1,6 +1,7 @@
 class JsitorTag < LiquidTagBase
   PARTIAL = "liquids/jsitor".freeze
   URL_REGEXP = /\A(https|http):\/\/(jsitor)\.(com)\/(embed)\/([a-zA-Z0-9]+)([?a-zA-Z&]*)\Z/.freeze
```

Comment:
I think this can be simplified to `\A(https|http):\/\/jsitor\.com\/embed\/\w+[?a-zA-Z&]*\Z`

you don't need grouping since you're just passing the link verbatim to the HTML